### PR TITLE
OCPBUGS#30070: Known issue with `opm` and RHEL 8

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.15

Issue: [OCPBUGS-30070](https://issues.redhat.com/browse/OCPBUGS-30070)

Link to docs preview:
* [Installing the opm CLI tool](https://72413--ocpdocs-pr.netlify.app/openshift-enterprise/latest/cli_reference/opm/cli-opm-install#olm-installing-opm_cli-opm-install) (NOTE: The URL in step 1 will resolve when the PR is merged)
* [Release notes > Known issues => (CTRL/CMD + F "opm CLI")](https://72413--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-known-issues)

QE review:
- [x] QE has approved this change.

Change management acks:
- [x] @sjstout (CS)
- [x] @kalexand-rh (DPM)
- [x] @kevinrizza (Engineering manager/Team lead)
- [x] @tlwu2013 (PM)
- [x] @xltian (QE)
- [x] @davemulford (PX)

### CODE REVIEW

File: snippets/opm-rhel8-known-issue.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Text snippet included in the following modules:
+//
+// * modules/olm-installing-opm.adoc
+// * release_notes/ocp-4-15-release-notes.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+There is currently a known issue where the version of the `opm` CLI tool released with {product-title} {product-version} does not support RHEL 8. As a workaround, RHEL 8 users can navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.14/[OpenShift mirror site] and download the latest version of the tarball released with {product-title} 4.14.
```
Comment (michaelryanpeter) [inline snippets/opm-rhel8-known-issue.adoc:8]: @grokspawn Do you have a Jira ticket for the RHEL8/ART work that I can link to at the end of this snippet?

**Comment by kalexand-rh** [comment]
LGTM

**Comment by sjstout** [comment]
LGTM

**Comment by davemulford** [comment]
LGTM

File: snippets/opm-rhel8-known-issue.adoc (hunk 1 above)
Comment (grokspawn) [reply snippets/opm-rhel8-known-issue.adoc:8]: Negative.  Will have one once we know what the container will look like.  It's already changed once.

**Review by grokspawn — APPROVED** [review]

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**